test/helpers/increaseTime.js

```diff
@@ -29,7 +29,7 @@ export default function increaseTime(duration) {
 }
 
 // sets time to given timestamp based on current block time
-export function setTimeTo(timestamp) {
+export async function setTimeTo(timestamp) {
```

Comment:
i added async there to make it clear that it's async ie. it returns a promise. I was a source of a bug in my previous tests